### ISSUE TITLE
Ctrl - and Ctrl _ don't work

### ISSUE THREAD

**Issue by samarism** [lihaoyi/Ammonite#528]
Is there a document which covers the set of keybindings for Ammonite REPL which work correctly for MAC? Ammonite is supposed to use a version of the "readline" editor, but some of those commands do not work, for example:

Delete only the current line **(CTRL K deletes all the input ahead of the cursor)**
Undo: **CTRL _ or CTRL - do not work?**

**Comment by lihaoyi** [comment]
`Ctrl k` is fixed by https://github.com/lihaoyi/Ammonite/pull/529

`Ctrl -` should work; it's defined [here](https://github.com/lihaoyi/Ammonite/blob/master/terminal/src/main/scala/ammonite/terminal/filters/UndoFilter.scala#L130). Not sure why it doesn't

**Comment by samarism** [comment]
On my Mac, "Ctrl -" does nothing. I asked about "Ctrl _" because the readline wiki page https://en.wikipedia.org/wiki/GNU_Readline seems to be using that for Undo. But in the ammonite documentation, "Ctrl -" is being used. Anyway, neither of them are undoing the last edit.

**Comment by NicolaeNMV** [comment]
I have the same problem with "Ctrl and -" (undo) on OSX Yosemite 10.10.3. 
"Ctrl and R" (reverse search) works fine.
After trying different variation of shell and terminals I pinpointed the regression to the amm version.

Works fine:
Welcome to the Ammonite Repl 0.5.7
(Scala 2.11.8 Java 1.8.0_51)

Not working:
Welcome to the Ammonite Repl 0.5.8
(Scala 2.11.8 Java 1.8.0_51)

"Ctrl + -" I've noticed that it creates white spaces ?

Welcome to the Ammonite Repl 0.5.9
(Scala 2.11.8 Java 1.8.0_51)

All version after 0.5.9 "Ctrl + -" doesn't produce any visible result.

Thanks by the way for great software.
